CONTRIBUTING.md

```diff
@@ -73,6 +73,26 @@ To update it, run the following command:
 bundle exec rubocop --auto-gen-config
 ```
 
+### Docker
+
+Alternatively you can use `Docker` to have your local development environment:
+
+Interactively:
+
+```bash
+docker-compose run --rm package bash
+
+root@1dc31cbe:/home/package# bundle install
+root@1dc31cbe:/home/package# bundle exec rspec
+root@1dc31cbe:/home/package# bundle exec rubocop -a
+```
+
+Or not:
+
+`docker-compose run --rm package bundle install`  
+`docker-compose run --rm package bundle exec rubocop -a`  
+`docker-compose run --rm package bundle exec rspec`  
```

Comment:
I don't understand how this will work. For me, it's launching 3 different containers not 3 commands in the same one no?

Reply:
Is each command a test to be done separately

Reply:
Yes, for example:

`docker-compose run --rm package bundle exec rubocop -a`  

Will start a container run the `bundle exec rubocop -a` command wait until the end (print the response in the STDOUT) and remove the container.
The other commands are the same idea.

This is the way I don't like to use it, that's why I added the other way above which run one container, and it will last until you close it, so you can run any command you want.

```bash
docker-compose run --rm package bash
root@1234# bundle exec rspec
root@1234# bundle exec rubocop -a
root@1234# bundle exec rspec
```

Reply:
I added both ways :)

Do you think it is good to remove them? And just add a message about the docker like `This project contains a docker-compose.yml configurated, you can use it to run all the commands mentioned below.` ?

Reply:
I also wonder what would be best. For me what would be good is the `docker-compose` and a `DockerFile` with all the test commands:
```
FROM ruby:2.6

WORKDIR /

ADD . .

RUN bundle install
RUN bundle exec rspec
RUN bundle exec rubocop -a
RUN bundle exec rspec
```
So you just have to run
```
docker-compose build
docker-compose up
```
And you can also run just `docker-compose run --rm package bash` but I don't think they both will work this way ...
I will really like it if the same file can allow to run interactively and just tests

Reply:
I would have added an `alternatively with docker` under every section. For example with the linter section: 

----- 

### Linter 

Each PR should pass the linter to be accepted.

```
# Check the linter errors
bundle exec rubocop lib/ spec/
# Auto-correct the linter errors
bundle exec rubocop -a lib/ spec/
```

Or with docker: 

```
docker-compose run --rm package bundle exec rubocop -a
```

----


I would have removed this part from the `tests` section

```
docker pull getmeili/meilisearch:latest # Fetch the latest version of Meilisearch image from Docker Hub
docker run -p 7700:7700 getmeili/meilisearch:latest meilisearch --master-key=masterKey --no-analytics
```

As there is no reason why it should be in this section. 
Put it in the `setup` category where we say, you can run the docker interactive mode or setup the env locally.


-----

I understand this implies a lot of changes in all the repos but it makes the more sense to me.

Reply:
We decided together in an internal meeting about adding a sentence right after the setup https://github.com/meilisearch/integration-guides/issues/199#issuecomment-1143576047